### PR TITLE
feat : Modal Component

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- A demo Modal component with actions.

### CODE REVIEW

File: Components/Modal/modal.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const btnAction = document.querySelector('#btn-actionModal');
+const btnInfo = document.querySelector('#btn-infoModal');
+const actionModal = document.querySelector('#actionModal');
+const closeModalBtn = document.querySelector('#closeModalBtn');
+
+const actionModalHandler = () => {
+    actionModal.classList.add("modalShow");
```
Comment (Hamdrive) [inline Components/Modal/modal.js:7]: You can also explore the use of toggle() property as well, having a click event listener to handle it

**Review by Hamdrive — COMMENTED** [review]
Nice implementation of modal, looks clean and simple
There is one additional suggestion apart from whats given below, try to add an overlay that doesnt allow any user mouse actions outside the modal

**Review by shuklahimanshu81 — APPROVED** [review]
Code structure is clean  and learnt how I can also use animation in my projects. Good work.

**Comment by sushmita12345** [comment]
- File structure is clean & easy to understand.
- Good use of variables.
- There is no breakpoints, you can proceed to merge.

**Review by sushmita12345 — APPROVED** [review]

**Pull request merged**